pyEpiabm/docs/source/routines.rst

```diff
@@ -4,15 +4,19 @@ Routines
 
 .. currentmodule:: pyEpiabm.routine
 
-Routines provides various methods to act upon or create a population.
+Routines provides various methods to act upon or create a population, or members of it.
 
 Overview:
 
-- :class:`CovidsimHelpers`
+- :class:`HouseholdForces`
```

Comment:
I feel HouseholdForces is a bit of a weird name. It suggests physical forces. Could we make FOI for force of infection well used and described throughout the code (a la covidsim) so HouseholdFOI and calc_household_foi etc, see below

Reply:
@Elizabeth-Hayman  Hmm this is the name for the class though, which includes other things too? Or were you saying that all the other factors that Ferguson later adds just modify the force of infection? I am certainly amenable to changing this!

Reply:
I decided to go for `HouseholdInfection` etc as the class (wanted to avoid using an acronym in the class name and thought we might want to extend the class later) - hope this is okay!

Reply:
Yep this is good